monkey/monkey_island/cc/models/monkey.py

```diff
@@ -68,6 +68,19 @@ def get_os(self):
             os = "windows"
         return os
 
+    @staticmethod
+    def get_tunneled_monkeys():
+        return Monkey.objects(tunnel__exists=True)
+
+    @staticmethod
+    def get_network_info(monkey):
```

Comment:
This shouldn't be static

Reply:
Agree, this should be a class method - like is_dead or get_os.